### PR TITLE
fix(android): show only the requested system bar

### LINKED ISSUE #8479: [Bug]: Android SystemBars.show({ bar: StatusBar }) also shows the navigation bar
### Capacitor Version

Current `main` (`e456de08`). I checked history as well: this behavior appears to have been present since the SystemBars plugin was introduced in `a32216ac` on November 12, 2025.

### Platforms Affected

- [ ] iOS
- [x] Android
- [ ] Web

### Current Behavior

On Android, `SystemBars.show({ bar: SystemBarType.StatusBar })` also shows the navigation bar.

The `hide()` path treats the two bars separately, but the `show()` path does not:

- `hide(StatusBar)` -> `WindowInsetsCompat.Type.statusBars()`
- `hide(NavigationBar)` -> `WindowInsetsCompat.Type.navigationBars()`
- `show(StatusBar)` -> `WindowInsetsCompat.Type.systemBars()`
- `show(NavigationBar)` -> `WindowInsetsCompat.Type.navigationBars()`

Since `systemBars()` includes both status and navigation bars, a status-only show request reveals both bars.

### Expected Behavior

`SystemBars.show({ bar: SystemBarType.StatusBar })` should only show the status bar, leaving the navigation bar unchanged.

### Reproduction

```ts
import { SystemBars, SystemBarType } from '@capacitor/core';

await SystemBars.hide({ bar: SystemBarType.NavigationBar });
await SystemBars.show({ bar: SystemBarType.StatusBar });
```

Expected:
The status bar appears and the navigation bar stays hidden.

Actual:
The navigation bar appears too.

### Why this looks like a bug

The public API/docs describe `bar` as "The system bar to hide or show", which implies `StatusBar` and `NavigationBar` should be handled independently.

The Android implementation already does that correctly in the `hide()` path, and iOS also behaves symmetrically. The Android `show()` path is the odd one out.

### Relevant code

`android/capacitor/src/main/java/com/getcapacitor/plugin/SystemBars.java`

Current `show()` branch:

```java
if (bar.isEmpty() || bar.equals(BAR_STATUS_BAR)) {
    windowInsetsControllerCompat.show(WindowInsetsCompat.Type.systemBars());
}
if (bar.isEmpty() || bar.equals(BAR_GESTURE_BAR)) {
    windowInsetsControllerCompat.show(WindowInsetsCompat.Type.navigationBars());
}
```

### Suggested fix

```java
if (bar.isEmpty()) {
    windowInsetsControllerCompat.show(WindowInsetsCompat.Type.systemBars());
} else if (bar.equals(BAR_STATUS_BAR)) {
    windowInsetsControllerCompat.show(WindowInsetsCompat.Type.statusBars());
} else if (bar.equals(BAR_GESTURE_BAR)) {
    windowInsetsControllerCompat.show(WindowInsetsCompat.Type.navigationBars());
}
```

### Additional Information

I did not find a recent commit that introduced this. `git blame` points back to the original SystemBars plugin implementation, so this looks like an original Android mapping bug rather than a regression from the recent inset work.

### PR DESCRIPTION
**Pull request by fallintoplace**
## Summary
- show only the requested Android system bar when `SystemBars.show({ bar })` is called
- add a focused Android unit test covering the `show()` mapping for the empty, status-bar, and navigation-bar cases

## Why
The Android `hide()` path already treats `StatusBar` and `NavigationBar` separately, but `show(StatusBar)` was calling `WindowInsetsCompat.Type.systemBars()`, which also reveals the navigation bar.

Fixes #8479.

## Testing
- `JAVA_HOME=/opt/homebrew/opt/openjdk@21 ANDROID_HOME=/Users/hoangvu/Library/Android/sdk ANDROID_SDK_ROOT=/Users/hoangvu/Library/Android/sdk PATH=/opt/homebrew/opt/openjdk@21/bin:$PATH ./gradlew -p capacitor testDebugUnitTest --tests com.getcapacitor.plugin.SystemBarsTest`

### CODE REVIEW

**Review by theproducer — APPROVED** [review]
@fallintoplace Thanks for the unit tests as well!

File: android/capacitor/src/main/java/com/getcapacitor/plugin/SystemBars.java [hunk 1]
```diff
@@ -297,10 +297,11 @@ private void setHidden(boolean hide, String bar) {
             return;
         }
 
-        if (bar.isEmpty() || bar.equals(BAR_STATUS_BAR)) {
+        if (bar.isEmpty()) {
             windowInsetsControllerCompat.show(WindowInsetsCompat.Type.systemBars());
```
Comment (OS-pedrogustavobilro) [inline android/capacitor/src/main/java/com/getcapacitor/plugin/SystemBars.java:318]: @theproducer should we propagate this equivalent logic to `hide` (above in this same `setHidden` method)? In general systemBars should just be statusBars+navigationBars, but maybe this could change in the future? But I suppose if it does then that probably means additional changes in this core plugin.

**Review by OS-pedrogustavobilro — APPROVED** [review]
